projects/plugins/jetpack/_inc/client/at-a-glance/blocks.jsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { getRedirectUrl } from '@automattic/jetpack-components';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import DashItem from 'components/dash-item';
+import { isFetchingPluginsData, isPluginActive } from 'state/site/plugins';
+
+const DashBlocks = props => {
+	const labelName = __( 'Jetpack blocks', 'jetpack' );
+	const support = {
+		text: __(
+			'Jetpack includes some blocks which can help you create your pages exactly the way you want them.',
+			'jetpack'
+		),
+		link: getRedirectUrl( 'jetpack-support-blocks' ),
+	};
+
+	if (
+		props.isFetchingPluginsData ||
+		props.isPluginActive( 'classic-editor/classic-editor.php' )
+	) {
+		return null;
+	}
+
+	return (
+		<DashItem label={ labelName } support={ support }>
+			<p className="jp-dash-item__description">
+				{ __(
+					'Jetpack blocks give you the power to deliver quality content that hook website visitors without needing to hire a developer or learn a single line of code.',
```

Comment:
```suggestion
					'Jetpack blocks give you the power to deliver quality content that hooks website visitors without needing to hire a developer or learn a single line of code.',
```
I think this may be a typo - 'content that hooks' would be what I'd expect to read.